### ISSUE TITLE
Please be aware that your account is STOLEN and ABUSED by the very platform!

### ISSUE THREAD

**Issue by jbluv** [DanevenChin/ssdlite_mobilenet_v2#3]
Hi! I'm also a user, or rather victim, of the very platform.

I noticed that the platform admin **stole and abused** your account to make money via Xianyu. Please check the those strange repo that **"you"** starred and forked:

![d1](https://user-images.githubusercontent.com/76879803/104671549-fff4d880-5718-11eb-87a1-21e43251fe82.jpg)

![d2](https://user-images.githubusercontent.com/76879803/104671561-0420f600-5719-11eb-8a61-cbe0ba31060c.jpg)

**Comment by MOGUIJOE** [comment]
Yes, I think GitStar.com.cn is stealing Personal Access Token from us!
